azurerm/internal/services/cosmos/resource_arm_cosmosdb_cassandra_keyspace.go

```diff
@@ -117,35 +117,35 @@ func resourceArmCosmosDbCassandraKeyspaceCreate(d *schema.ResourceData, meta int
 		return fmt.Errorf("Error making get request for Cosmos Cassandra Keyspace %s (Account %s): %+v", name, account, err)
 	}
 
-	id, err := azure.CosmosGetIDFromResponse(resp.Response)
-	if err != nil {
-		return fmt.Errorf("Error retrieving the ID for Cosmos Cassandra Keyspace '%s' (Account %s) ID: %v", name, account, err)
+	if resp.ID == nil {
+		return fmt.Errorf("Error getting ID from Cosmos Cassandra Keyspace %s (Account %s)", name, account)
```

Comment:
_minor_ if we use `%q` rather than `%s` these'll get quoted too:

```suggestion
		return fmt.Errorf("Error getting ID from Cosmos Cassandra Keyspace %q (Account %q)", name, account)
```